Fst2.h

```diff
@@ -157,6 +157,8 @@ struct fst2Tag {
      * tokens, and compound words will be handled via a compound pattern.
      */
     int compound_pattern;
+    bool stop;
```

Comment:
It would be necessary to provide documentation for this variable.

Reply:
done